index.php

```diff
@@ -63,19 +87,37 @@
   if (isset($_POST["entry_1922375458"])) { //organisations
     $requested_orgs = filter_var_array($_POST["entry_1922375458"], FILTER_SANITIZE_STRING);
     foreach ($requested_orgs as $requested_org) {
-      if (in_array($requested_org, $allowed_orgs) && !empty($requested_org) ) { //!!!!FIX ME!!!!
+      if (in_array($requested_org, $allowed_orgs) && !empty($requested_org) ) { //!!!!FIX ME!!!!I would but I don't know how you're broken good sir...
         $orgs[] = $requested_org;
       }
     }
   }
-
+  //print_r($allowed_orgs);
+  if (isset($_POST["transaction_provider_org"])) { //organisations
+    $requested_orgs = filter_var_array($_POST["transaction_provider_org"], FILTER_SANITIZE_STRING);
+    foreach ($requested_orgs as $requested_org) {
+      if (in_array($requested_org, $allowed_orgs) && !empty($requested_org) ) { //!!!!FIX ME!!!!I would but I don't know how you're broken good sir...
```

Comment:
See [this comment](https://github.com/IATI/IATI-Query-Builder/pull/71/files#r140635099). I think it’s safe to remove this FIX ME.